rust/connlib/libs/tunnel/src/tun_darwin.rs

```diff
@@ -168,7 +168,7 @@ impl IfaceDevice {
     }
 
     pub fn name(&self) -> Result<String> {
-        let mut tunnel_name = [0u8; 256];
+        let mut tunnel_name = [0u8; IF_NAMESIZE];
```

Comment:
Figured we should use the constant here.

Reply:
Which is actually only 16 characters it appears: https://github.com/apple/darwin-xnu/blob/main/bsd/net/if.h#L69